street_network/src/lanes/tests.rs

```diff
@@ -0,0 +1,211 @@
+use abstutil::Tags;
+use geom::Distance;
+
+use crate::{get_lane_specs_ltr, Direction, DrivingSide, MapConfig};
+
+// osm2lanes has a more extensive unit test suite, so why does this one exist? This also checks the
```

Comment:
I decided to keep these unit tests long term. They serve a different purpose than the more detailed ones in osm2lanes